src/sentry/integrations/slack/webhooks/event.py

```diff
@@ -150,7 +150,7 @@ def on_link_shared(self, request: Request, slack_request: SlackDMRequest) -> boo
                 return True
 
             # Don't unfurl the same thing multiple times
-            seen_marker = hash(json.dumps((link_type, args), sort_keys=True))
```

Comment:
Is this a relic from python 2?

Reply:
yeah so the kwargs are being dropped on the floor in `json.py` -- I think there was definitely _intent_ to use `sort_keys=True` here but it does nothing so I removed it